packages/UI/src/components/organisms/Dropdown/StyledDropdown.ts

```diff
@@ -43,7 +43,7 @@ export const OptionsContainer = styled.div<DropdownProps>`
 
 export const Options = styled.form<DropdownProps>`
   display: flex;
-  width: 100%;
+  width: 75%;
```

Comment:
Instead of updating the `width` here, could we pass in `'sm'` or `'md'` as the `width` prop to the `Dropdown`?

This change will affect all instances of Dropdown components which may have some unintended effects.

Reply:
We actually are passing in a width prop to the dropdown. I chose "lg" because there's a lot of available space above the idea cards on the BrowseIdeas page. I added a change to the width of the options themselves because many of the options were extending beyond the dropdown container div. Maybe we could find a different approach to only change the width from 100% under certain conditions?

Reply:
@jordanmcousins how about we remove this line and remove 
```
label {
    width: 100%;
  }
```
all together

Reply:
That makes sense, I'll implement that change